assets/src/components/conversations/ConversationsContainer.tsx

```diff
@@ -169,6 +170,24 @@ class ConversationsContainer extends React.Component<Props, State> {
     await this.refreshSelectedConversation();
   };
 
+  handleDeleteConversation = async (conversationId: string) => {
+    await this.props.onDeleteConversation(conversationId);
+
+    notification.open({
+      message: 'Conversation deleted!',
+      duration: 2, // 2 seconds
+      description: (
+        <Box>
+          You can view this conversations once again{' '}
+          <a href="/conversations/all">here</a>.
```

Comment:
hmm is this true? if the conversation gets deleted, i don't think it should show up anywhere anymore, right?

Reply:
ahh sorry copy paste error